turtlebot2_drivers/CMakeLists.txt

```diff
@@ -35,12 +35,20 @@ ament_target_dependencies(dumb_teleop
   "rclcpp"
   "geometry_msgs")
 
-install(TARGETS kobuki_node dumb_teleop
-        DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
+install(TARGETS
+  kobuki_node
+  dumb_teleop
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin)
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
 endif()
 
+install(
+  PROGRAMS launch/turtlebot_carto_2d.py launch/turtlebot_carto_3d.py
```

Comment:
The reason I ended up not doing that is that there are two "old" launch files in that directory that we may not want to install.  Of the two, turtlebot_follow.py definitely doesn't make any sense; it is referencing the "follower" node, which we aren't building anymore.  The turtlebot_joy.py could be made to work with modifications.  What do you think we should do here?